install_config/install/deploy_router.adoc

```diff
@@ -269,9 +280,63 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
 ====
 
 
+[[working-with-multiple-routers]]
+=== Working With Multiple Routers
+
+Multiple routers can be grouped to distribute routing load in the cluster and separate
+tenants to different routers.  Each router in the group shardes a set of routes that are
+tagged using labels. Router sharding over the whole cluster uses ROUTE_LABELS.
+Sharding over a namespace (project) uses NAMESPACE_LABELS.
+
+
+Because each router, by default, uses ports 80 (http) and 443 (https) the routers must be
+placed on nodes in the cluster that are not already using those ports, or the ports must
+be changed using the ROUTER_SERVICE_HTTP_PORT and ROUTER_SERVICE_HTTPS_PORT environment
+variables. When namespace labels are used, the service account for the router must have
+`cluster-reader` permission.
+
+[[adding-noedselector-to-a-deployment]]
+=== Adding a Node Selector to a Deployment Config
+
+Make specific routers deploy on specific nodes by labeling the nodes and
+adding a node selector to the router deployment controller.
+
+Add a label to the desired node.
+====
+----
+$ oc label node 10.254.254.28 "router=first"
+----
+====
+Don't forget to restart the node when the config is changed.
+
+Add a node selector to the deployment configuration.
+====
+----
+$ oc edit dc <deploymentConfigName>
+----
+====
+Find the section “spec” under “template” and add node selector section
+and add the nodeSelector.
+====
+----
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        router: router1
+    spec:
+      nodeSelector:
+        router: "first"
+----
+====
+When the edit is saved the deployment is updated.
+
 [[creating-router-shards]]
 === Creating Router Shards
 
+Router sharding distributes routes among a set of routers. As above the routers are
+created on different nodes.
```

Comment:
This "as above" doesn't consider the different port case... and I don't understand why this paragraph is even here... it doesn't seem to add anything new.